src/radiation_system.hpp

```diff
@@ -998,12 +1011,63 @@ AMREX_GPU_HOST_DEVICE auto RadSystem<problem_t>::ComputeEgasFromEint(const doubl
 	return Etot;
 }
 
+// This function is not used.
```

Comment:
What is this function for?

Reply:
This is a function I wrote for the IMEX scheme to balance the time evolution in hydro and radiation. Later I found a better way to do it and this function is abandoned. I'll delete it.